spec.html

```diff
@@ -4030,7 +4991,7 @@ <h1>ToString ( _argument_ )</h1>
               Number
             </td>
             <td>
-              Return NumberToString(_argument_).
+              Return Number::toString(_argument_).
```

Comment:
`!` or `?`?

Reply:
It should be `!`, since we already know that `argument` is a Number.